modules/microshift-storage-vol-snapshot-restore.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="restoring-vol-snapshot-microshift_{context}"]
+= Restoring a volume snapshot
+
+The following workflow demonstrates snapshot restoration. In this example, the verification steps are also given to ensure that data written to a source `PersistentVolumeClaim` (PVC) is preserved and restored on a new PVC.
+
+[IMPORTANT]
+====
+A snapshot must be restored to a PVC of exactly the same size as the source volume of the snapshot. You can resize the PVC after the snapshot is restored successfully if a larger PVC is needed.
+====
+
+.Procedure
+
+. Restore a snapshot by specifying the `VolumeSnapshot` object as the `dataSource` in a `PersistentVolumeClaim` by using the following example:
```

Comment:
I think you can use the generic object references for data source and PVC. If it is important to use the specific API references, then they need to be followed with a noun.

```suggestion
. Restore a snapshot by specifying the `VolumeSnapshot` object as the data source in a persistent volume claim by entering the following command:
```